app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -65,8 +66,17 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
     showProfile = profileClick;
     showSettings = settingsClick;
 
-    creatorDashboardButtonIsGone = environment.currentConfig().observable()
-      .map(config -> !coalesce(config.features().get(FeatureKey.ANDROID_CREATOR_VIEW), false));
+    final Observable<Boolean> userIsCreator = currentUser.observable()
+      .map(u -> u != null && u.createdProjectsCount() > 0);
```

Comment:
we have a `currentUser.loggedInUser()` Observable (in our [CurrentUserType](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/CurrentUserType.java#L68)) that has the null filter built in which you can use so we can consolidate this logic to something like:

```java
currentUser.loggedInUser()
  .map(u -> IntegerUtils.isNonZero(u.createdProjectsCount()));
```
The `IntegerUtils.isNonZero()` func is also useful here because `createdProjectsCount` is a `@Nullable` field on the User, and this ensures we won't get a NPE!